environments/lfric/spack.yaml

```diff
@@ -1,8 +1,10 @@
 spack:
   specs:
-    - xios@2.5.2252 +lfric
-    - yaxt@0.9.0 idxtype=long
+    - xios@2.5.2629 +lfric
+    - yaxt@0.11.3 idxtype=long
     - pfunit@3 max_array_rank=6 +mpi +openmp
+    - tau +mpi +openmp +opari ~pthreads +lfric
```

Comment:
I will try that soon :)

Reply:
This is a straight copy of what Jason has in the internal container, so should work just the same